iothub/service/src/IotHubSasCredentialProperties.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.Amqp;
+
+#if !NET451
+
+using Azure;
+
+#endif
+
+namespace Microsoft.Azure.Devices
+{
+    internal class IotHubSasCredentialProperties : IotHubConnectionProperties
+    {
+#if NET451
+
+        public IotHubSasCredentialProperties()
+        {
+            throw new InvalidOperationException("nameof(AzureSasCredential) is not supported in NET451");
```

Comment:
Should the error be "IotHubSasCredentialProperties is not supported in NET451" instead?

Reply:
I think it should be IotHubSasCredential is not supported in NET451. The IotHubSasCredentialProperties class is internal but the constructors will now expose IotHubSasCredential  instead of AzureSasCredential.